gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -219,10 +218,11 @@ protected void submitFlowToDagManagerHelper(String flowGroup, String flowName) {
   protected void createMetrics() {
     super.messagesRead = this.getMetricContext().counter(RuntimeMetrics.DAG_ACTION_STORE_MONITOR_PREFIX + "." + RuntimeMetrics.GOBBLIN_KAFKA_HIGH_LEVEL_CONSUMER_MESSAGES_READ);
     super.queueSizeGauges = new ContextAwareGauge[super.numThreads];
-    for (int i=0; i < numThreads; i++) {
+    for (int i = 0; i < numThreads; i++) {
       // An 'effectively' final variable is needed inside the lambda expression below
       int finalI = i;
       this.queueSizeGauges[i] = this.getMetricContext().newContextAwareGauge(
+          RuntimeMetrics.DAG_ACTION_STORE_MONITOR_PREFIX + "." +
```

Comment:
couldn't we just define a method:
```
protected String getMetricsPrefix()
```
and have `super.createMetrics()` call that to make this minor adjustment, rather than basically copying and pasting the def w/ only a slight change?

Reply:
Made the changes you described above